lib/runPrettier.js

```diff
@@ -0,0 +1,61 @@
+const { spawn } = require('child_process');
+const path = require('path');
+
+const cwd = process.cwd();
+
+const filePattern = 'src/**/*.js';
+
+function runPrettier({ write, listDifferent, config }) {
+  const prettierConfig = config || {};
+
+  const prettierPath = path.join(
+    path.resolve('node_modules/prettier'),
```

Comment:
In this situation path.resolve will resolve to an absolute path from the cwd.

https://nodejs.org/docs/latest/api/path.html#path_path_resolve_paths